src/animation.cpp

```diff
@@ -1,24 +1,29 @@
 #include "animation.h"
 
+#include <time.h>
+
 #include "avatar.h"
+#include "character.h"
+#include "creature.h"
+#include "cursesdef.h"
+#include "explosion.h"
 #include "game.h"
+#include "game_constants.h"
+#include "input.h"
 #include "map.h"
+#include "memory_fast.h"
 #include "monster.h"
 #include "mtype.h"
 #include "options.h"
 #include "output.h"
 #include "player.h"
+#include "point.h"
 #include "popup.h"
```

Comment:
Required for MSVC builds.
```suggestion
#include "popup.h"
#include "posix_time.h"
```